### PR TITLE
Add section about json-ld authoring and processing

### PR DESCRIPTION
**Pull request by mattgarrish**
This PR probably isn't ready to merge at this stage, but looking for feedback about whether it begins to address the proposal in #11?

On the right track? Needs improvement?

(I haven't touched on removing base, and I grouped the first three sections under the intro banner as we were getting a lot of preparatory sections in a row.)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pub-manifest/pull/30.html" title="Last updated on Aug 12, 2019, 1:37 PM UTC (4bbe095)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pub-manifest/30/48c82a8...4bbe095.html" title="Last updated on Aug 12, 2019, 1:37 PM UTC (4bbe095)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -133,55 +133,74 @@ <h3>Terminology</h3>
...
+				</section>
 
-				<p>This part of the specification describes the requirements for creating an authored manifest,
-					regardless of the format of the digital publication. Rules for constructing a canonical manifest
-					from the authored manifest are defined for each specific implementation.</p>
+				<section id="manifest-jsonld">
+					<h4>JSON-LD Authoring and Processing</h4>
 
+					<p>The manifest format enables a number of shorthands to the [[!json-ld]] syntax to simplify
+						authoring manifests. It is not always required that object types be explicitly authored, for
```
Comment (iherman) [inline index.html]: This is misleading. We do not have "shorthands to the json-ld syntax": we have, in effect, a simple and more complex or, shall we say, a simple and a complete set of information (essentially a Graph), both of them valid JSON-LD, and a transformation that converts the simple graph to the complete one.

(I am not sure how to express this in a consise way.)

File: index.html [hunk 2]
```diff
@@ -133,55 +133,74 @@ <h3>Terminology</h3>
...
+					<p>As a result, the graph generated from an authored manifest might not always be the same as the
+						graph generated from the canonical manifest. If consistency is desired, it is RECOMMENDED to
+						avoid these shorthands in the authored manifest.</p>
+
+					<p>Similarly, it is OPTIONAL for a user agent to be a full JSON-LD processor. User agents only need
+						to be able to read the manifest and internalize the data following the processing steps defined
+						in this specification. As a result, use of [[!json-ld]] features not described in this
+						specification (e.g., <code>@base</code>, JSON literals) is not advised, as user agents might
+						ignore such featuers or process them incorrectly.</p>
+				</section>
```
Comment (iherman) [inline index.html]: featuers -> features

**Comment by iherman** [comment]
@mattgarrish 

> I haven't touched on removing base

I am not sure there is anything to be removed. The general notion of "base", ie, as inherited from the HTML file or the URL of the manifest file, is perfectly fine and we should keep that. What is _not_ mentioned is the possibility for the author to add a `"@base":"http://...."` as part of the manifest (it is a bit as we were saying that the author could not use the `<base ...>` in the HTML file).

File: index.html (hunk 1 above)
Comment (mattgarrish) [reply index.html]: What about: "The manifest format enables information to be compactly expressed to simplify authoring manifests."

**Comment by mattgarrish** [comment]
> I am not sure there is anything to be removed.

Okay. I didn't read back through the thread to see if this was a request to bring it in or if we had something lurking somewhere like in the processing steps.

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html]: Actually... shouldn't we say "The authored manifest enables information to be ..." ? Just to make it clear that this is what the authored manifest is about?

Comment (mattgarrish) [reply index.html]: Yes, good catch. I've also linked to the definitions of authored and canonical manifest.

**Comment by iherman** [comment]
(I am not sure why but the 'preview' does not seem to reflect the latest changes on the text...:-(

**Comment by iherman** [comment]
https://raw.githack.com/w3c/pub-manifest/editorial/add-jsonld-rel/index.html reflects the latest changes

**Comment by mattgarrish** [comment]
Things have been a bit odd lately. I had a commit not get added to an open PR the other day, even though it was shown when I switched to the branch. I ended up having to recommit the changes after merging. Looks like it's not that problem again, though.

File: index.html [hunk 3]
```diff
@@ -133,55 +133,74 @@ <h3>Terminology</h3>
...
+
+					<p>As a result, the information represented by the authored manifest might be less complete than the
+						one represented by the <a>canonical manifest</a>. If consistency between the two states is
+						desired, it is RECOMMENDED to avoid the compact forms in the authored manifest.</p>
+
+					<p>Similarly, it is OPTIONAL for a user agent to be a full JSON-LD processor. User agents only need
+						to be able to read the manifest and internalize the data following the processing steps defined
+						in this specification. As a result, use of [[!json-ld]] features not described in this
+						specification (e.g., <code>@base</code>, JSON literals) is not advised, as user agents might
+						ignore such features or process them incorrectly.</p>
```
Comment (BigBlueHat) [inline index.html]: JSON-LD features "not described in this specification" won't be easy for implementors to pin down...

It's probably simpler (and more consistent with other JSON-LD-based specifications) to instead reference the JSON serialization/structure that must be used to be considered a valid "publication manifest document" (enforced by via the spec text and a non-normative JSON Schema perhaps).

[ActivityStreams](https://www.w3.org/TR/activitystreams-core/#syntaxconventions) is a great example of this approach:
> This specification describes a JSON-based [RFC7159] serialization syntax for the Activity Vocabulary that conforms to a subset of [JSON-LD] syntax constraints but does not require JSON-LD processing. While other serialization forms are possible, such alternatives are not discussed by this document.

**Comment by mattgarrish** [comment]
Closing this pr as I'm going to restart the changes needed.

**Pull request closed without merging**